interfaces/mount/backend.go

```diff
@@ -58,7 +58,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, confinement interfaces.ConfinementO
 	}
 	content := deriveContent(spec.(*Specification), snapInfo)
 	// synchronize the content with the filesystem
-	glob := fmt.Sprintf("%s.fstab", interfaces.SecurityTagGlob(snapName))
+	glob := fmt.Sprintf("snap.%s.*fstab", snapName)
```

Comment:
We cannot do that: the `*` captures both the `snap.SNAP.APP.fstab` and `snap.SNAP.fstab`. The extra dot would prevent that from working and this is indeed why the pattern is changed by this PR.